ext/standard/basic_functions.c

```diff
@@ -3646,6 +3659,8 @@ PHP_MINIT_FUNCTION(basic) /* {{{ */
 # endif
 #endif
 
+	BASIC_MINIT_SUBMODULE(random)
```

Comment:
The prototype to this needs to be somewhere in the header, otherwise there'll be visibility issue.